components/brave_rewards/resources/page/reducers/wallet_reducer.ts

```diff
@@ -91,19 +84,11 @@ const walletReducer: Reducer<Rewards.State | undefined> = (state: Rewards.State,
       break
     }
     case types.ON_GET_EXTERNAL_WALLET: {
-      const { value, error } = action.payload.result
-
-      if (value) {
-        state = { ...state, externalWallet: value.wallet }
+      const { externalWallet } = action.payload
+      if (externalWallet) {
+        state = { ...state, externalWallet }
         chrome.send('brave_rewards.fetchBalance')
-      } else {
-        switch (error) {
-          case mojom.GetExternalWalletError.kAccessTokenExpired:
-            chrome.send('brave_rewards.getExternalWallet')
-            break
```

Comment:
Was about to ask if we won't miss this, but as I see, we no longer have anything in the "get external wallet" flow that would require `kAccessTokenExpired`.